x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -483,7 +483,21 @@ export const AlertsList: React.FunctionComponent = () => {
     },
     {
       field: 'executionStatus.lastExecutionDate',
-      name: 'Last run',
+      name: (
```

Comment:
Can/should we add a test for this?

Reply:
Test added in https://github.com/elastic/kibana/pull/120159/commits/b2408b8eb38b25b3e3b8cd79d78a0e5357a28573!